src/website/views/components/drawer/_examples.pug

```diff
@@ -217,19 +354,43 @@ h3 Titled
         a(href='#html-titled') HTML Blueprint
   .chi-tabs-panel.-active#webcomponent-titled
     :code(lang="html")
-      <chi-drawer position="left" title="Drawer title here" backdrop active>
+      <!-- Trigger -->
```

Comment:
Behavior is slightly inconsistent in all examples. The HTML Blueprint code dismisses the drawer when the backdrop is clicked. The Web Component code does not.